WordPress/src/main/res/values-en-rAU/strings.xml

```diff
@@ -66,7 +66,7 @@ Language: en_AU
 	<string name="button_not_now">Not now</string>
 	<string name="media_error_timeout">Server response timed out, you may have a poor connection or server error</string>
 	<string name="media_error_internal_server_error">Upload error. Try enabling Optimise Images in Site->Settings</string>
-	<string name="media_error_too_large_upload">File too large to upload. Try enabling Optimise Images in Site->Settings</string>
+	<string name="media_error_http_too_large_upload">File too large to upload. Try enabling Optimise Images in Site->Settings</string>
```

Comment:
Looks like these non-English strings need to be updated to match the English strings from the last commit.